### ISSUE TITLE
Editing of text inside Vizy blocks limited (Firefox)

### ISSUE THREAD

**Issue by gglnx** [verbb/vizy#58]
**Description**

I've created a Vizy block with a default Plain Text field (with enabled line breaks). It's not possible to select any text inside this text field (not by click and drag and not by double click). It's also not possible to position the text marker anywhere into the field. Navigation is only possible with the arrow keys.

This is the same case for Vizy fields inside a Vizy block. The only exception: Double click to select everything works, but it's not possible to de-select.

**Steps to reproduce**
1. Create a Plain Text field
2. Create a Vizy field with a block containing the Plain Text field
3. Create an entry with this Vizy field, try to select any text

**Additional info**
- Plugin version: 1.0.3
- Craft version: 3.6.17
- Multi-site: No

**Comment by engram-design** [comment]
Out of interest, what browser are you using? I can't replicate this on Chrome, and I know there's some issues with Safari. Is this also nested in any Matrix fields?

Just in case my setup is different to yours: 

https://user-images.githubusercontent.com/1221575/125940833-f483b2fa-ef65-41aa-9d84-2f2bf1a63fe6.mp4

**Comment by gglnx** [comment]
I'm using Firefox.

**Comment by gglnx** [comment]
No, nested Matrix fields. The same setup as you.

**Comment by engram-design** [comment]
Ah, you're right - looks like a Firefox issue. Looking into it.

**Comment by engram-design** [comment]
Just following it up with Tiptap https://github.com/ueberdosis/tiptap/issues/1668

**Comment by engram-design** [comment]
Looks like this is a Firefox [browser issue](https://bugzilla.mozilla.org/show_bug.cgi?id=739071), rather than Tiptap or Prosemirror. You can see this illustrated here - http://jsfiddle.net/crawf101/7s9x51f6/4/

**Comment by engram-design** [comment]
Managed to come up with a fix for the next release. To get the fix early, change your `verbb/vizy` requirement in `composer.json` to:

```
"require": {
  "verbb/vizy": "dev-craft-3 as 1.0.8",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.0.9](https://github.com/verbb/vizy/releases/tag/1.0.9)
